bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/entity/SharingState.kt

```diff
@@ -35,8 +35,8 @@ class SharingState(
     var externalId: String,
 
     @Enumerated(EnumType.STRING)
-    @Column(name = "business_partner_type", nullable = false)
-    var businessPartnerType: BusinessPartnerType,
+    @Column(name = "business_partner_type", nullable = true)
+    var businessPartnerType: BusinessPartnerType? = BusinessPartnerType.NULL,
```

Comment:
Ok, here you made it nullable already. So the additional NULL enum value is not needed